.github/workflows/publish.yaml

```diff
@@ -0,0 +1,28 @@
+name: build-and-publish
+
+on:
+  push:
+    tags: [ "*" ]
+  workflow_dispatch:
+
+permissions:
+  id-token: write
+  contents: read
+  packages: write
+
+defaults:
+  run:
+    # We need -e -o pipefail for consistency with GitHub Actions' default behavior
+    shell: bash -e -o pipefail {0}
+
+
+jobs:
+  publish-package:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest] #windows-latest?
```

Comment:
same as above, don't think macos makes sense right now.

Reply:
100% its removed in the steps that are presently running

Reply:
removed